values.yaml

```diff
@@ -35,6 +35,9 @@ global:
   # hostname: the external hostname of the ingress, this may be the cluster master or public load-balancer
   hostname: localhost
 
+  # If specified is the service name to use typically in a piepline this will be the
```

Comment:
I'm not sure this comment makes sense. I don't know what you're trying to say here.